webapp/src/shared/components/Input/EditUserForm.tsx

```diff
@@ -2,39 +2,85 @@ import { useState } from 'react';
 import { Button, ButtonVariant, Input, InputVariant } from '../';
 import './EditUserForm.css';
 
-export default function EditUserForm() {
-  const [fullName, setFullName] = useState('');
-  const [email, setEmail] = useState('');
+type EditUserProps = {
+  origFullName: string;
+  origEmail: string;
+};
+
+export default function EditUserForm({
```

Comment:
Just like we have the generic wrapper around fetch for GET requests that is the hook UseData, maybe we could have a generic wrapper around other request types? Maybe no...

Reply:
Maybe instead we can make generic the EditUserForm itself, into EditForm, by passing from the parent component, the request options, the url endpoint, and the placeholders ... ?
Or maybe it's not to be made generic. Not sure.

Reply:
We can talk about it. Anyway I going to double-check what we can do

Reply:
And I'll create an issue in order not to forget it